plugins/auth-backend/README.md

```diff
@@ -7,6 +7,16 @@ This is the backend part of the auth plugin.
 It responds to auth requests from the frontend, and fulfills them by delegating
 to the appropriate provider in the backend.
 
+## Requirements
+
+Needs AUTH_GOOGLE_CLIENT_ID and AUTH_GOOGLE_CLIENT_SECRET set in the environment for the backend to startup
+
+## Local development
+
+export AUTH_GOOGLE_CLIENT_ID=<INSERT_CLIENT_ID_HERE>
+read -r AUTH_GOOGLE_CLIENT_SECRET=<INSERT_CLIENT_ID_HERE>
```

Comment:
hmmm maybe i'm misunderstanding, but did you mean something like `export AUTH_GOOGLE_CLIENT_SECRET=$(<  CLIENT_ID_FILE)` or so? or just remove the part after `=` if you meant that the user should enter it interactively

Reply:
You need to `export AUTH_GOOGLE_CLIENT_SECRET` afterwards as well, or it will only be available directly in the shell